cmd/dep/graphviz.go

```diff
@@ -91,3 +91,24 @@ func (dp gvnode) label() string {
 
 	return strings.Join(label, "\n")
 }
+
+// Ensure that the literal string prefix is a path tree match and
+// guard against possibilities like this:
+//
+// github.com/sdboyer/foo
+// github.com/sdboyer/foobar/baz
+//
+// Verify that either the input is the same length as the match (in which
+// case we know they're equal), or that the next character is a "/". (Import
+// paths are defined to always use "/", not the OS-specific path separator.)
+func isPathPrefixOrEqual(pre, path string) bool {
```

Comment:
Now that I'm looking at this here, I realize the change in parameter order is terrible. Let's flip them so that it's the same as strings.HasPrefix() - string first, prefix second.